docs/NSR.md

```diff
@@ -223,7 +220,7 @@ CA Infrastructure MUST be in a Physically Secure Environment.
 CA Infrastructure and Network Equipment MUST be implemented and configured to authenticate and encrypt connections:
 
    1. between CA Infrastructure components; and
-   2. between CA Infrastructure and non-CA Infrastructure.
+   2. between CA Infrastructure and non-CA Infrastructure under CA management.
```

Comment:
I remember we discussed this during the call, but upon reflection I am not sure this change really helps to clear it up.

As I remember, the motivation is that it may not be possible to encrypt "connections", and I believe we gave the example of DNS queries. In this specific example, but also more generally, clearly we can expect Infrastructure under CA management that it will not be possible to encrypt connections to, either.